sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -663,6 +665,80 @@ def process(self, context):
     self.check_label(beam.ParDo(MyDoFn()), r'ParDo(MyDoFn)')
 
 
+class PTransformTestDisplayData(unittest.TestCase):
+  def test_map_named_function(self):
+    ########################
```

Comment:
Now that these are separate tests, do these block comments add any value? If not, remove.